src/main/scala/com/rbmhtechnology/calliope/StoredEvent.scala

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2015 - 2017 Red Bull Media House GmbH <http://www.redbullmediahouse.com> - all rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.rbmhtechnology.calliope
+
+import java.time.Instant
+
+case class StoredEvent(sequenceNr: Long, creationTimestamp: Instant, topic: String, aggregateId: String, event: Array[Byte])
```

Comment:
Ok, thanks for clarifying. Anyway let's discuss that in a later design session. For now this looks good.